cmd/controller/app/server.go

```diff
@@ -278,6 +283,18 @@ func Run(o *options.ControllerRunOptions, ctx context.Context) error {
 	go func() {
 		<-mgr.Elected()
 
+		kubeInfomers.Start(ctx.Done())
```

Comment:
consider moving the informer.Start outside the goroutine, because when the mgr stars running, it needs the informer, right? so putting it ahead of `mgr.Start` is better in my opinion